pipelines/salesforce/__init__.py

```diff
@@ -0,0 +1,142 @@
+"""Pipeline for Salesforce depending on the simple_salesforce python package.
+
+Imported resources are: account, campaign, contact, lead, opportunity, pricebook_2, pricebook_entry, product_2, user and user_role
+
+Salesforce api docs: https://developer.salesforce.com/docs/apis
+
+To get the security token: https://onlinehelp.coveo.com/en/ces/7.0/administrator/getting_the_security_token_for_your_salesforce_account.htm
+"""
+
+import typing as t
+import pendulum
+
+from typing import Any, Dict, Iterator, Iterator, Optional
+
+import dlt
+from simple_salesforce import Salesforce
+
+
+def is_production() -> bool:
+    """Return True if the current environment is production.
+
+    You can override this function based on your own setup. It will
+    add a LIMIT 100 clause to the queries.
+    """
+    return True
+
+def get_records(
+    sf: Salesforce,
+    sobject: str,
+    last_state: t.Optional[dict] = None,
```

Comment:
this is last value, not last state, and is not a dict

please add `py.typed` to your pipeline folder so mypy will check the typing. see other pipelines for example

Reply:
Done, why do we not enforce linting by default over the entire repo?